docs/products/postgresql/howto/troubleshooting-connection-pooling.rst

```diff
@@ -0,0 +1,25 @@
+Troubleshooting connection pooling 
+==================================
+
+PgBouncer is a lightweight connection pooler for PostgreSQL® with low memory requirements (2 kB per connection by default).
+
+PgBouncer offers several methods when rotating connections:
+
+Session pooling
+  Most polite method. When a client connects, a server connection will be assigned to it for the whole duration it stays connected. When the client disconnects, the server connection will be put back into pool. This mode supports all PostgreSQL features.
+
+Transaction pooling
+  A server connection is assigned to a client only during a transaction. When PgBouncer notices that the transaction is over, the server will be put back into the pool.
+  This mode breaks a few session-based features of PostgreSQL. You can use it only when the application cooperates by not using features that break. See the table below for incompatible features.
```

Comment:
.. warning::
   
   This mode breaks a few session-based features of PostgreSQL. Use it only when the application cooperates without using the features that break.

> See the table below for the incompatible features.

What table do you mean?